userguide/content/en/docs/Updating/Convert-site-to-module.md

```diff
@@ -171,32 +168,47 @@ hugo: cleaned module cache for "github.com/twbs/bootstrap"
 ```
 {{% /alert %}}
 
-### Clean up your repository
+## Clean up of your repository
 
-Since your site now uses Hugo Modules, your previously used `themes` directory can be removed: 
+Since your site now uses Hugo Modules, you can clean up your repo now.
+At the command prompt, change to the root directory of your existing site first.
 
 ```
-rm -rf /path/to/your/theme
+cd /path/to/my-existing-site
+```
+
+### Previous use of Docsy theme as git clone
+
+Simply remove the subdirectory ´docsy` inside your `themes` directory: 
+
+```
+rm -rf themes/docsy
+```
+
+### Previous use of Docsy theme as git submodule
+
+If your Docsy theme was installed as submodule, you need to unregister the `docsy` submodule from your repo first:
+
+```
+git submodule deinit themes/docsy/
 ```
```

Comment:
Curious, I don't recall having had to `deinit` a submodule before. Hmm, from the [git docs](https://git-scm.com/docs/git-submodule), they suggest using `git rm`:

> If you _really_ want to remove a submodule from the repository and commit that use [git-rm[1]](https://git-scm.com/docs/git-rm) instead.

Reply:
> `deinit` removes the relevant submodule line from `.gitmodules`.

AFAIR, `git rm` does that too, no? (I'll test that tomorrow.)

Reply:
> > `deinit` removes the relevant submodule line from `.gitmodules`.
> 
> AFAIR, `git rm` does that too, no? (I'll test that tomorrow.)

You are right here. I removed the `deinit` command from the description since it is not needed.